pandas/tseries/offsets.py

```diff
@@ -231,18 +230,21 @@ def __add__(date):
         - microsecond
         - nanosecond
 
+        .
+
     See Also
     --------
-    dateutil.relativedelta.relativedelta
+    dateutil.relativedelta.relativedelta :     The relativedelta type is designed to be applied to an existing datetime an can replace specific components of that datetime, or represents an interval of time.
```

Comment:
You'll have to split the description in different lines, this is surely too long. Also, leave just one space after the colon.